polkadot-protocol/protocol-components/cryptography.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Cryptography
+description: A concise guide to cryptography in blockchain, covering hash functions, encryption types, digital signatures, and elliptic curve applications.
+---
+
+# Cryptography
+
+## Introduction
+
+Cryptography forms the backbone of blockchain technology, providing the mathematical verifiability crucial for consensus systems, data integrity, and user security. While a deep understanding of the underlying mathematical processes isn't necessary for most blockchain developers, grasping the fundamental applications of cryptography is essential. This page offers a comprehensive overview of various cryptographic implementations used across Polkadot SDK and the broader blockchain ecosystem.
+
+## Hash Functions
+
+Hash functions are fundamental to blockchain technology, creating a unique digital fingerprint for any piece of data including simple text, images, or any other form of file. They map input data of any size to a fixed-size output (typically 32 bytes) using complex mathematical operations. Hashing is used for verifying data integrity, creating digital signatures, and providing a secure way to store passwords. This form of mapping is known as the 'pigeonhole principle', and it is primarily implemented to efficiently and verifiably identify data from large sets.
+
+### Key Properties of Hash Functions
+
+- **Deterministic** - the same input always produces the same output
+- **Quick computation** - it's easy to calculate the hash value for any given input
+- **Pre-image resistance** - it's infeasible to generate the input data from its hash
+- **Small changes in input yield large changes in output** - known as the avalanche effect
+- **Collision resistance** - it's extremely difficult to find two different inputs with the same hash
```

Comment:
IMO, I'd suggest tweaking this line a bit. Instead of "extremely difficult," I'd say "the probabilities are extremely low." But the decision is up to you